src/components/FeatureViews/Availability/OOOConfirmation.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import checkMarkIcon from "components/FeatureViews/Scheduler/icons/bluecheckmark.svg"
+import { Typography, Grid, Button } from "@material-ui/core/";
+import { omouBlue } from "../../../theme/muiTheme";
+import { makeStyles } from "@material-ui/core/styles";
+
+
+
+const useStyles = makeStyles((theme) => ({
+    messageSent: {
+        fontSize: '30px',
+        color: "#43B5D9",
+        fontWeight: "bold"
+    },
+    messageContext: {
+        fontSize: "16px",
+
+    },
+    cancelbutton: {
+        backgroundColor: "#FFFFFF"
+
+    },
+
+}));
+
+
+
+export default function OOOConfirmation(props) {
```

Comment:
destructure handleClose from props since it's the only thing you're pulling from it

Reply:
bump